src/components/Title.js

```diff
@@ -8,10 +8,10 @@ export const Title = styled.Text`
   padding: 10px;
   text-transform: uppercase;
 `;
-export const TitleContainer = styled.View`
-  box-shadow: 0 6px 6px black;
-`;
 
 //experimentig with the shadow ,
 //it seams that titles on homepage are inside of a box witch has a shadow
 // I need to recrate the effect of having a shadow from the line
+export const TitleContainer = styled.View`
+  box-shadow: 0 6px 6px black;
```

Comment:
for shadows maybe we can try to have a single <Shadow /> styled component, that we add after containers which needs a shadow.

example:
```
<TitleContainer>
  <Title>asdasd<Title>
</TitleContainer>
<Shadow />
<...>
```

Reply:
Shadow is still failing, but I created a fake shadow effect for now with border.